### ISSUE TITLE
MySql.Data.MySqlClient.MySqlException

### ISSUE THREAD

**Issue by xtings** [ONLYOFFICE/CommunityServer#329]
Dear OnlyOffice:

After the installation, I followed instructions and changed customerror in Web.config and I received:

Access denied for user 'onlyoffice_user'@'onlyoffice-community-server.onlyoffice' (using password:YES)
Description: HTTP500. Error processing request.
Details: Non-web exception. Exception origin (name of application or object): MySql.Data.

I read previous issues and found I can use 

'bash communityserver-configure.sh'

However, I cannot find communityserver-configure.sh. When I entered docker container and ran the command, I received

'bash: communityserver-configure.sh: No such file or directory.'

How can I deal with the error? Thanks.

**Comment by Carazyda** [comment]
Hello @xtings Please descirbe in more details how did you run Onlyoffice? If you use docker please look at this [repo](https://github.com/ONLYOFFICE/Docker-CommunityServer).

**Comment by xtings** [comment]
> Hello @xtings Please descirbe in more details how did you run Onlyoffice? If you use docker please look at this [repo](https://github.com/ONLYOFFICE/Docker-CommunityServer).

**Comment by xtings** [comment]
> Hello @xtings Please descirbe in more details how did you run Onlyoffice? If you use docker please look at this [repo](https://github.com/ONLYOFFICE/Docker-CommunityServer).


Yes, I used docker to run the community server. I followed the instructions as you mentioned, however, I still met the problem I described in the issue. 

**Access denied for user 'onlyoffice_user'@'onlyoffice-community-server.onlyoffice' (using password:YES)
Description: HTTP500. Error processing request.
Details: Non-web exception. Exception origin (name of application or object): MySql.Data.**

Thank you.

**Comment by Carazyda** [comment]
please describe in more details how did you run mysql server and connect it to community server?

**Comment by xtings** [comment]
> please describe in more details how did you run mysql server and connect it to community server?

I followed these commands in https://github.com/ONLYOFFICE/Docker-CommunityServer

echo "[mysqld]
sql_mode = 'NO_ENGINE_SUBSTITUTION'
max_connections = 1000
max_allowed_packet = 1048576000
group_concat_max_len = 2048
log-error = /var/log/mysql/error.log" > /app/onlyoffice/mysql/conf.d/onlyoffice.cnf

echo "CREATE USER 'onlyoffice_user'@'localhost' IDENTIFIED BY 'onlyoffice_pass';
CREATE USER 'mail_admin'@'localhost' IDENTIFIED BY 'Isadmin123';
GRANT ALL PRIVILEGES ON * . * TO 'root'@'%' IDENTIFIED BY 'my-secret-pw';
GRANT ALL PRIVILEGES ON * . * TO 'onlyoffice_user'@'%' IDENTIFIED BY 'onlyoffice_pass';
GRANT ALL PRIVILEGES ON * . * TO 'mail_admin'@'%' IDENTIFIED BY 'Isadmin123';
FLUSH PRIVILEGES;" > /app/onlyoffice/mysql/initdb/setup.sql

sudo docker run --net onlyoffice -i -t -d --restart=always --name onlyoffice-mysql-server \
 -v /app/onlyoffice/mysql/conf.d:/etc/mysql/conf.d \
 -v /app/onlyoffice/mysql/data:/var/lib/mysql \
 -v /app/onlyoffice/mysql/initdb:/docker-entrypoint-initdb.d \
 -e MYSQL_ROOT_PASSWORD=my-secret-pw \
 -e MYSQL_DATABASE=onlyoffice \
 mysql:5.7

sudo docker run --net onlyoffice -i -t -d --privileged --restart=always --name onlyoffice-community-server -p 80:80 -p 443:443 -p 5222:5222 \
 -e MYSQL_SERVER_ROOT_PASSWORD=my-secret-pw \
 -e MYSQL_SERVER_DB_NAME=onlyoffice \
 -e MYSQL_SERVER_HOST=onlyoffice-mysql-server \
 -e MYSQL_SERVER_USER=onlyoffice_user \
 -e MYSQL_SERVER_PASS=onlyoffice_pass \
 -v /app/onlyoffice/CommunityServer/data:/var/www/onlyoffice/Data \
 -v /app/onlyoffice/CommunityServer/logs:/var/log/onlyoffice \
 -v /app/onlyoffice/CommunityServer/letsencrypt:/etc/letsencrypt \
 -v /sys/fs/cgroup:/sys/fs/cgroup:ro \
 onlyoffice/communityserver

sudo docker run -i -t -d --privileged -p 8080:80 \
-v /app/onlyoffice/CommunityServer/logs:/var/log/onlyoffice \
-v /app/onlyoffice/CommunityServer/data:/var/www/onlyoffice/Data \
-v /app/onlyoffice/CommunityServer/letsencrypt:/etc/letsencrypt \
-v /sys/fs/cgroup:/sys/fs/cgroup:ro onlyoffice/communityserver

**Comment by Carazyda** [comment]
So why did you run communityserver twice? It seems that the second container is not connected to the mysql server.
